src/main/java/io/getunleash/metric/UnleashMetricServiceImpl.java

```diff
@@ -28,6 +37,14 @@ public UnleashMetricServiceImpl(
         this.started = LocalDateTime.now(ZoneId.of("UTC"));
         this.unleashConfig = unleashConfig;
         this.metricSender = metricSender;
+        this.maxInterval =
+                Integer.max(
+                        20,
+                        300
```

Comment:
Is maxInterval in seconds? What are these magic numbers for? 

I'm struggling to see why the largest sendMetricsInterval is, the smaller maxInterval becomes. Or it's too late for math atm.

Ok, on second thought, is it the number of times we want to skip/avoid making an HTTP call? If that's the case I'd suggest renaming the variable to `skip` and `maxSkips` maybe, as Interval has a different meaning in the java.time API which might mislead readers of this code

Reply:
Good point, I'll rename. Yes, it is the number of times we're going to skip/do nothing.